src/ex_docmd.c

```diff
@@ -6923,6 +6923,111 @@ ex_wrongmodifier(exarg_T *eap)
     eap->errmsg = ex_errmsg(e_invalid_command_str, eap->cmd);
 }
 
+#ifdef FEAT_EVAL
+/*
+ * Skip to the next newline character.
+ */
+    static char_u *
+skiptonewline(char_u *p)
+{
+    while (*p && *p != NL)
```

Comment:
Can this not use something like `memchr`? But see earlier comment about not parsing newline-delimited filenames